assets/xml/objects/object_kz.xml

```diff
@@ -1,93 +1,106 @@
+        <!-- This is the same skin texture as below, but this one is her arms and back -->
+        <Texture Name="gKaizokuSkinGradient1Tex" OutName="skin1" Format="ci8" Width="8" Height="8" Offset="0x5AD0" TlutOffset="0x58D0" />
+        <Texture Name="gKaizokuShirtTopRedGradientTex" OutName="shirt_burgundy" Format="ci8" Width="8" Height="8" Offset="0x5B10" />
+        <!-- Gold armor plates on her shin, forearms, belt. Similar texture as Emblem, but slightly darker. -->
+        <Texture Name="gKaizokuArmor1Tex" OutName="armor" Format="ci8" Width="16" Height="16" Offset="0x5B50" />
+        <Texture Name="gKaizokuShirtTopPatternAndVeilTex" OutName="shirt_pattern1" Format="i8" Width="16" Height="16" Offset="0x5C50" />
+        <Texture Name="gKaizokuBellyButtonTex" OutName="navel" Format="ci8" Width="16" Height="16" Offset="0x5D50" />
+        <Texture Name="gKaizokuPantsTex" OutName="pants" Format="i8" Width="16" Height="16" Offset="0x5E50" />
+        <!-- This is the texture at the back of her boots and pants belt center.  -->
+        <Texture Name="gKaizokuZipper1Tex" OutName="zipper1" Format="i8" Width="8" Height="16" Offset="0x5F50" />
 
-        <DList Name="object_kz_DL_007CB0" Offset="0x7CB0" />
-        <DList Name="object_kz_DL_008130" Offset="0x8130" />
-        <DList Name="object_kz_DL_008268" Offset="0x8268" />
-        <DList Name="object_kz_DL_008490" Offset="0x8490" />
-        <DList Name="object_kz_DL_0085C8" Offset="0x85C8" />
-        <DList Name="object_kz_DL_0087F0" Offset="0x87F0" />
-        <DList Name="object_kz_DL_008A10" Offset="0x8A10" />
-        <DList Name="object_kz_DL_008B10" Offset="0x8B10" />
-        <DList Name="object_kz_DL_008D80" Offset="0x8D80" />
-        <DList Name="object_kz_DL_008E80" Offset="0x8E80" />
+        <DList Name="gKaizokuTorsoDL" Offset="0x7CB0" />
+        <DList Name="gKaizokuRightUpperArmDL" Offset="0x8130" />
+        <DList Name="gKaizokuRightForarmDL" Offset="0x8268" />
+        <DList Name="gKaizokuLeftUpperArmDL" Offset="0x8490" />
+        <DList Name="gKaizokuLeftForarmDL" Offset="0x85C8" />
+        <DList Name="gKaizokuWaistDL" Offset="0x87F0" />
+        <DList Name="gKaizokuRightThighDL" Offset="0x8A10" />
+        <DList Name="gKaizokuRightShinDL" Offset="0x8B10" />
+        <DList Name="gKaizokuLeftThighDL" Offset="0x8D80" />
+        <DList Name="gKaizokuLeftShinDL" Offset="0x8E80" />
 
-        <Texture Name="object_kz_TLUT_009100" OutName="tlut_009100" Format="rgba16" Width="252" Height="1" Offset="0x9100" />
-        <Texture Name="object_kz_Tex_0092F8" OutName="tex_0092F8" Format="ci8" Width="8" Height="8" Offset="0x92F8" />
-        <Texture Name="object_kz_Tex_009338" OutName="tex_009338" Format="ci8" Width="8" Height="16" Offset="0x9338" />
+        <Texture Name="gKaizoku2TLUT" OutName="kaizoku_tlut2" Format="rgba16" Width="4" Height="63" Offset="0x9100" /> 
```

Comment:
Would probably just remove `kaizoku` from the outname since everything else you've done doesn't include it (note there are other outnames this would apply to).